pdf_report/mailer.py

```diff
@@ -0,0 +1,165 @@
+"""Script for sending the PDF reports as email attachments using SES."""
+
+import logging
+import os
+from os import environ as ENV
+from datetime import datetime
+from email.mime.application import MIMEApplication
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+import boto3
+import botocore.exceptions
+from botocore.exceptions import ClientError
+from dotenv import load_dotenv
+import psycopg2
+import psycopg2.extras
+from psycopg2.extensions import connection as DBConnection, cursor as DBCursor
+
+load_dotenv()
```

Comment:
Disagree with this being here, why not main?

Reply:
It doesn't need to be in the main block as all it does is load the environment variables. Furthermore, I have a global constant that uses an environment variable. All in all, I don't think it matters too much